ptx/S_12_6_Curl.ptx

```diff
@@ -1,106 +1,469 @@
 <section xml:id="S_12_6_Curl">
   <title>The Curl of a Vector Field</title>
-  
-    <objectives>
-      <ul>
+<objectives>
+  <ul>
+    <li>
+      <p>
+      What is meant by rotation of a vector field in a plane?
+      </p>
+    </li>
+    <li>
+      <p>
+        How can we generalize our two dimensional measurement of rotation to work in three dimensions?
+      </p>
+    </li>
+    <li>
+      <p>
+        How can you measure the rotational strength of a vector field?
+      </p>
+    </li>
+  </ul>
+</objectives>
+<introduction>
+  <p>
+    In <xref ref="S_12_5_Div">Section</xref>, we examined how the strength of a vector field in two (or more) dimensions changed in different regions. In particular, we developed the <xref ref="Div_defn">divergence (Definition</xref>) of a vector field as a local (or density) measurement for how the strength of the vector field changes.
```

Comment:
Try `<xref ref="Div_defn" text="custom">divergence</xref>` instead of the current `xref` to the definition of divergence here. That would make a knowl in HTML for someone who wants to see the definition and a link in PDF. I think it's safe to assume enough competence on the part of a print reader that they could find the definition of divergence without having the number for it. If you do want the word Definition and the number to appear, I think there's a way to get that effect with a different value for `@text`. (In the Green's Theorem section, I'm using `@text="title"` when I create an `xref` to Green's Theorem so then the clickable for the known is just "Green's Theorem". (That works because I put `title` inside `theorem`.)